NBitcoin/BIP158/GolombRiceFilter.cs

```diff
@@ -290,6 +290,38 @@ private static ulong SipHash(ulong k0, ulong k1, byte[] data)
 			hasher.Write(data);
 			return hasher.Finalize();
 		}
+
+		#region EqualityAndComparison
+
+		public override bool Equals(object obj) => obj is GolombRiceFilter tx && this == tx;
```

Comment:
The variable name for filter shouldn't be `tx` but `filter` or any other better name.

Reply:
fixed